src/index.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+import * as yargs from 'yargs';
+import { initProject as createProject } from './create-project';
+import { openDocs as docs } from './docs';
+import { test } from './test';
+
+yargs
+  .usage('Usage: kak-cli <command>')
+  .version()
+  .command(
+    ['create-project <dir> [version]','cp <dir> [version]'], 
+    'Create a new kakunin project in directory with optional kakunin version, default latest',
+    (args: yargs.Argv) => {
+
+      return yargs
+        .option('dir', {
+          demandOption: true,
+          string: true,
+        })
+        .option('version', {
+          demandOption: false,
+          string: true,
+        });
+    },
+    (argv) => {
+      console.log(`Starting to create new kakuknin project. This will take a while.`)
+      createProject(argv.dir)(argv.version);
+    },
+  )
+  .command(['docs', 'd', 'documentation'],'Go to the documentation at https://kakunin.io',(): any => {}, docs)
+  .command(['test', 't'],'Testing command', (): any => {}, test)
```

Comment:
Any reason for this?

Reply:
Nope, unnecessary testing command - deleted.